### PR TITLE
feat(use-query-params): add ability to push instead of replacing current history

### PR DESCRIPTION
**Pull request by chambo-e**

### CODE REVIEW

**Review by philibea — APPROVED** [review]

File: packages/use-query-params/README.md [hunk 1]
```diff
@@ -86,3 +86,28 @@ const Component = () => {
...
+
+To avoid mutating history
+
+```js
+// In this exemple we assume that we have an URL that include : `?company=Scaleway".
+import React from 'react'
+import useQueryParams from '@scaleway/use-query-params'
+
+const Component = () => {
+  const { queryParams, setQueryParams } = useQueryParams()
```
Comment (adriengibrat) [inline packages/use-query-params/README.md:100]:
```suggestion
  const { queryParams, replaceQueryParams } = useQueryParams()
```

Comment (chambo-e) [reply packages/use-query-params/README.md:100]: We can use both :)

**Review by adriengibrat — APPROVED** [review]

File: packages/use-query-params/README.md (hunk 1 above)
Comment (adriengibrat) [reply packages/use-query-params/README.md:100]: example use `replaceQueryParams` just below

Comment (chambo-e) [reply packages/use-query-params/README.md]: Right, sorry my bad :)

**Pull request merged**